tianshou/utils/logger/wandb.py

```diff
@@ -156,3 +160,8 @@ def restore_data(self) -> tuple[int, int, int]:
         except KeyError:
             env_step = 0
         return epoch, env_step, gradient_step
+
+    def restore_logged_data(self, log_path: str) -> dict:
+        # TODO: add support for restoring logged data from W&B
+        assert self.tensorboard_logger is not None
```

Comment:
replace by value error or remove optionality of the field

Reply:
done